README.md

```diff
@@ -10,6 +10,14 @@ author: '@everyone'
 
 Kumpulan _resource_ untuk Pemrosesan bahasa alami Bahasa Indonesia. Repository ini dibuat untuk membantu _Researcher_/Mahasiswa yang sedang membuat tugas akhir atau penelitian tentang pemrosesan bahasa alami yang menggunakan Bahasa Indonesia.
 
+Group of resources for Natural Language Processing in Indonesia language. This repository is created to helps reseracher or student who doing task about final thesis or research about natural language processing using Indonesia language.
```

Comment:
I think "A collection of" is better than "Group of resources" wdyt?
and I think is better to keep "Bahasa Indonesia" than "Indonesia language"

Reply:
Agree it at all, wait I'll fix it

Reply:
fixed yakk @raisoturu